lib/jwt/jwt.go

```diff
@@ -200,6 +200,29 @@ func (k *Key) SignSnowflake(p SignParams, issuer string) (string, error) {
 	return k.sign(claims)
 }
 
+type PROXYSignParams struct {
+	ClusterName        string
+	SourceAddress      string
+	DestinationAddress string
+}
+
+const expirationPROXY = time.Second * 60
+
+// SignPROXY will create short lived signed JWT that is used in signed PROXY header
+func (k *Key) SignPROXY(p PROXYSignParams) (string, error) {
+	claims := Claims{
+		Claims: jwt.Claims{
+			Subject:   fmt.Sprintf("%s/%s", p.SourceAddress, p.DestinationAddress),
```

Comment:
Nit: I'd have a short function for producing the subject, just so we can prevent it drifting between the signing and verification side. It'll also provide us a place that'll be the source of truth.

Reply:
Added 👍